playbooks/common/openshift-cluster/openshift_hosted.yml

```diff
@@ -27,21 +27,21 @@
       logging_elasticsearch_ops_cluster_size: "{{ openshift_hosted_logging_elasticsearch_ops_cluster_size | default(1) }}"
   roles:
   - role: openshift_hosted
-  - role: openshift_hosted_metrics
+  - role: openshift_metrics
```

Comment:
i dont think you meant to modify this file yet, not until QE has certified both new roles